jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -17,15 +17,31 @@ pipeline {
 
         stage('Clone Operator Repo') {
             steps {
-                //ToDo Clone the integeratly-operator repo
-                echo "Clone Operator Repo"
+                cleanWs()
```

Comment:
This should prob be higher up actually so im not wiping anything that happens earlier

Reply:
It needs to one of the first things that you do alright, but i think here is fine tbh. There should be nothing happening before this that is doing anything in the workspace anyway.